external/fv3viz/fv3viz/_plot_cube.py

```diff
@@ -448,7 +387,7 @@ def _validate_cube_shape(lat_shape, lon_shape, latb_shape, lonb_shape, array_sha
         raise ValueError("Lon, lat, and data_var each must be 3-dimensional.")
 
 
-def plot_cube_axes(
+def _plot_cube_axes(
```

Comment:
I couldn't find usage of this outside of notebooks, and my understanding is that everything you'd want to use this for can be done with plot_cube or pcolormesh_cube.